public/config/routes.rb

```diff
@@ -62,8 +68,10 @@
     get "repositories/:rid/classifications/:id/tree/node_from_root" => 'classifications#tree_node_from_root'
 
     #CLASSIFICATION TERMS
-    get  "repositories/:repo_slug/classification_terms/:slug_or_id" => 'classifications#term'
-    get  "classification_terms/:slug_or_id" => 'classifications#term'
+    get "repositories/:repo_slug/classification_terms/:slug_or_id" => 'classifications#term'
```

Comment:
@andrew-morrison Is there a reason this route is not included in the following line's `if` scope?

Reply:
Because if it were then there wouldn't be any route to the `'classifications#term` controller. Perhaps it should be, and then a route that specifies `:rid` and `:id` params should be added?

Reply:
This doesn't really have anything to do with these particular changes, but I'm actually a little perplexed about how this particular route was/is working in the first place. It looks like this is the only route pointing to `classifications#term` and it is passing the params as `:repo_slug` and `:slug_or_id`. Sure enough, Rails logs the following when processing the request:
`Processing by ClassificationsController#term as HTML`
`Parameters: {"repo_slug"=>"2", "slug_or_id"=>"1"}`

If you look at the `term` method in `classifications_controller.rb` it grabs `params[:rid]` and `params[:id]`, which is what made me wonder why it wasn't crashing. However, if I log `params` first thing in the `term` method, I get: 
`{"controller"=>"classifications", "action"=>"term", "repo_slug"=>"2", "slug_or_id"=>"1", "id"=>"1", "rid"=>"2"}`. 
@andrew-morrison Do you happen to have any idea how `id` and `rid` are getting plugged in there? So far I haven't been able to track it down. Is this some sort of Rails dark magic I'm not aware of? 🤔😅

Reply:
My understanding is that the `process_slug_or_id` method determines whether a `slug_or_id` parameter is either a slug or an id, and looks up the latter if it is the former, for routes which are ambiguous. In the case of classifications, that is triggered by this:

https://github.com/archivesspace/archivesspace/blob/0628775f0042eb44508d3c5d256a729d19562c28/public/app/controllers/classifications_controller.rb#L7-L9

Reply:
Ah, that's it, thanks!